src/mpid/ch3/channels/nemesis/netmod/ofi/ofi_tag_layout.h

```diff
@@ -30,7 +30,7 @@
 #define MPID_PGID_SHIFT          (32)
 #define MPID_PSOURCE_SHIFT       (0)
 #define MPID_PORT_SHIFT          (0)
-#define MPID_TAG_BITS            (28)
+#define MPID_OFI_TAG_BITS        (28)
```

Comment:
For consistency, can you prefix all the constants here with `MPID_OFI_` in a patch before this one?